css/owned-card.styl

```diff
@@ -58,8 +58,28 @@
       background: lighten(COBOLT_BLUE, 20%)
 
 .collection-card
+  -webkit-border-radius: 1em
+  border-radius: 1em
+
   .name
     margin-bottom: 1em
+    height: 2.9em
+    overflow: hidden
+    text-overflow: ellipsis
+    display: -webkit-box
+    -webkit-box-orient: vertical
+    -webkit-line-clamp: 2
```

Comment:
Isn't `display: -webkit-box` and the `box` syntax deprecated anyway? [caniuse.com](http://caniuse.com/) is a great reference for this, also noted as deprecated on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/box-flex) too.